### ISSUE TITLE
When using OIDC protocol, the "exp" claim of the JWT which is generated at login cannot be set to have validity less than 1 hour

### ISSUE THREAD

**Issue by stefanzahariaeg** [keycloak/keycloak#32001]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

oidc

### Describe the bug

I'm using OIDC integration and our client is willing to have 5 minutes expiry time for the token which is generated.
So, we configured in Realm -> Clients -> Our client -> Advanced -> Advanced Settings, all the tokens from there to "Expire in" 5 minuntes.
Then we did a login, and we got the bearer token that was generated.
We checked the claims from the JWT, and the "exp" claim was still set to iat + 1 hour.
We tried multiple combinations, also tried to see if anybody else raised this issue but didn't find any explanation.

### Version

24.0.5

### Regression

- [ ] The issue is a regression

### Expected behavior

After changing the lifespan of tokens, exp claim in the JWT should be updated

### Actual behavior

After changing the lifespan of tokens, exp claim is not updated

### How to Reproduce?

Have a OIDC integration set up, between an application and keycloak. Do a login and get a token from keycloak, then analyze the token

**Comment by rmartinc** [comment]
@stefanzahariaeg You are talking about the access token and in common code flow, right?

In that scenario the option you have to set is `Access Token Lifespan` and set to `Expired in` and set to 5 minutes. Doing a test in 24.0.4 and in the default account console I receive the correct exp of 5 minutes (just chaging that property in the account-console client).

```json
{
  "exp": 1723104363,
  "iat": 1723104063,
  "auth_time": 1723104063,
  "jti": "b49ae165-9dc1-4c12-96cf-1122cbb79d9a",
  "iss": "http://localhost:8080/realms/master",
  ...
}
```

Besides it's very weird you have one hour because that's not the default either.

In summary, I cannot reproduce in 24.0.4 or current 25.0.2. Can you test in a fresh 25.0.2?
